docs/external-template-creation.md

```diff
@@ -1,46 +1,35 @@
+- The **SVGs** used in each template are stored in [`assets/src/edit-story/stickers/`](https://github.com/google/web-stories-wp/tree/main/assets/src/edit-story/stickers) (in the `main` branch).
+- The (non-SVG) **image &amp; video files** used in each template are stored in [`public/static/main/images/templates/`](https://github.com/google/web-stories-wp/tree/static-site/public/static/main/images/templates) (in the `static-site` branch, using [Git LFS](https://git-lfs.github.com/)).
 
-To get the raw story json, simply open up a story in the web story editor with the devtools network tab open and press `save`. You should see the the story's json show up in the network tab. In that json, right click the `story_data` field and click `Store as global variable`. Go to the devtools console and type 
+## Overview
 
-```javascript
-copy(temp1)
-```
-
-When you press enter the story data should now be copied to your clipboard and ready to paste where you need.
-
-**Note**`Command+Shift+Option+J / Control+Shift+Alt+J` in the story editor opens up a dialog you can copy/paste raw story json to/from. However this is missing some fields `story_data` contains which are necessary for versioning the external templates.
-
-## Formatting Media Names In Story Creation
-
-WordPress has a naming convention where if you upload an image with the same name multiple times, it adds a `-x` suffix. 
-
-**i.e**
-if I upload `some_image.png` 3 times to the WordPress media upload in the story editor, it will store those images as:
-
-```javascript
-// some_image.png
-// some_image-1.png
-// some_image-2.png
-```
-
-Because of this, our conversion script will strip the `-x` suffix off of the image source so it can point to the correct image stored in our repo.
+To add a new template to the editor:
 
-This is an important note to make when naming images for upload into a story as **a `-x` suffix will always be stripped**. This naming convention is reserved for WordPress and will break your template creation if you use it in your image naming.
-
-## Adding Your Template
-
-Once you've added your template's raw story json to `packages/templates/src/raw/<template_name>.json`, you can import it into `packages/templates/src/getTemplates` and add to the function `loadTemplates(imageBaseUrl)`. It should then be accessible in `packages/templates/src/index`.
+1. [Engineer] Commit all SVGs used in the template to the codebase as stickers ([details](#adding-svgs-to-the-codebase-as-stickers)).
+2. [Designer] Create a new story in your local WP environment and replicate the template design.
+    - SVGs should already be available in the Stickers panel from step (1) &mdash; click on a sticker to insert it into the story. If you can't find the Stickers Panel, make sure the `enableStickers` experiment is turned on.
+    - Upload (drag & drop) images and videos into the story to add them.
```

Comment:
Should we ddo this on staging? or some other dedicated public endpoint vs local?

Reply:
Doesn't really matter I suppose.

On a public site you could collaborate with others during the process, so that's probably preferable.

Reply:
Only reason I'd advocate for doing it on a public endpoint is that a designer could just point a dev to a story vs being responsible for capturing the story data

Reply:
Yeah I suppose that somewhat falls under collaboration :-)

We could add a paragraph like `Note: if you are working on a template with other people, it might be a good idea to create the template on a public site where you can share access.` if we want to.